scripts/prepublish

```diff
@@ -25,3 +25,4 @@ done
 tsc
 
 cp -r types dist/
+rm -rf dist/test
```

Comment:
I am not sure this is required. 
Did you try to configure it through 'package.json'?

Reply:
well, it can't be done with package.json - but it can be done with .npmignore: its an ignore file with all the rules of .gitignore, so we can include/exclude at will - but the language is reversed...